datalad/support/gitrepo.py

```diff
@@ -635,99 +636,178 @@ def _parse_progress_line(self, line):
         return True
 
 
-class GitPythonProgressBar(RemoteProgress):
-    """A handler for Git commands interfaced by GitPython which report progress
+class StdOutCaptureWithGitProgress(GitProgress):
+    proc_out = True
+
+
+class FetchInfo(dict):
+    """
+    dict that carries results of a fetch operation of a single head
```

Comment:
Based on diffing with git.remoteFetchInfo, I see this is stripped down/modified quite a bit, but I still think this should reference the GitPython implementation as GitProgress's docstring does.  Same probably goes for PushInfo too.

Reply:
Will do.

Reply:
Done.